onadata/apps/logger/models/instance.py

```diff
@@ -90,6 +91,28 @@ def update_xform_submission_count(sender, instance, created, **kwargs):
         )
 
 
+def update_user_submissions_counter(sender, instance, created, **kwargs):
+    if not created:
+        return
+    if getattr(instance, 'defer_counting', False):
+        return
```

Comment:
Not sure if it is needed.

Reply:
I tested it with or without. Submissions are likely to be counted twice without it